ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh

```diff
@@ -158,9 +162,12 @@ check_result=0
 echo "------check enabled capabilities-----"
 echo "enabled capability set: ${enabled_capability_set}"
 for cap in $enabled_capability_set; do
-    if [[ "${cap}" == "Build" ]] || [[ "${cap}" == "DeploymentConfig" ]]; then
+    echo "check capability ${cap}"
+    if [[ "${caps_resource_list}" =~ ${cap} ]]; then
```

Comment:
```suggestion
    if [[ " ${caps_resource_list} " =~ " ${cap} "  ]]; then
```

Reply:
maybe we can try this to see if it make `shellcheck` happy.

Reply:
shellcheck complains it
```
^------^ SC2076: Remove quotes from right-hand side of =~ to match as a regex rather than literally.
```
I disabled shellcheck for this line.